src/ngraph_encapsulate_op.cc

```diff
@@ -310,6 +291,12 @@ class NGraphEncapsulateOp : public OpKernel {
         << "NGraphEncapsulateOp::Compute got freshness tracker for cluster "
         << m_ngraph_cluster;
 
+    // Get the backend.
+    std::shared_ptr<ng::runtime::Backend> backend;
+    bool is_cpu_backend;
+    std::tie(backend, is_cpu_backend) =
+        BackendManager::Instance()->GetBackend();
+
```

Comment:
the `BackendManager::Instance()->GetBackend()` should be in a try-catch

Reply:
Done